src/transaction/transaction_manager.cpp

```diff
@@ -43,8 +43,8 @@ void TransactionManager::Abort(TransactionContext *const txn) {
   table_latch_.Lock();
   const timestamp_t start_time = txn->StartTime();
   size_t ret UNUSED_ATTRIBUTE = curr_running_txns_.erase(start_time);
-  TERRIER_ASSERT(ret == 1, "aborted transaction did not exist in global transactions table");
-  if (gc_enabled_) completed_txns_.push(txn);
+  TERRIER_ASSERT(ret == 1, "Aborted transaction did not exist in global transactions table");
```

Comment:
Similar change to prev. TERRIER_ASSERT, if possible.

Reply:
Similar to previous. We need the erase() call in Release mode.